app/views/layouts/career_site/_footer.html.slim

```diff
@@ -1,4 +1,13 @@
-footer.pt-2.px-4
+footer.pt-2.pb-3.px-4
```

Comment:
There's a hole under the footer and the content inside overflows the container
This is because the `footer-min-height` variable is set to `height` instead of `min-height` for some reason
![image](https://github.com/user-attachments/assets/3acd8e88-71d3-4917-bbdb-8b3c2506c3b3)

Reply:
By the way, why do we need a fixed height for the footer and wrapper?